test/dice_test.coffee

```diff
@@ -0,0 +1,27 @@
+Helper = require('../node_modules/hubot-test-helper/src/index.coffee')
+
+helper = new Helper('../scripts/dice.coffee')
+
+expect = require('chai').expect
```

Comment:
The only case I can make is that Chai provides BDD assertions inline with Mocha. That doesn't mean I can justify it as I don't give a fuck about BDD.